assets/react/components/cta-box.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from 'styled-components'
+import PropTypes from "prop-types";
+
+function CtaBox({title, description, image, backgroundColor}) {
+  return (
+    <Wrapper style={{
+        '--background-color': backgroundColor
+    }}>
+      {image && <Img src={image} alt=""/>}
+      <div>
+        <Title>{title}</Title>
+        {description && <Description>{description}</Description>}
+      </div>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  position: fixed;
+  max-width: calc(((100vw / var(--grid-columns))*2) - calc(var(--border-width) * 6));
+  bottom: calc(var(--border-width) * 3);
+  left: calc(var(--border-width) * 3);
+  background-color: var(--background-color, hsl(0deg 0% 100%));
+  border: calc(var(--border-width) * 2) solid var(--background-color);
+  filter: var(--filter-shadow);
+  display: flex;
+  column-gap: calc(var(--border-width) * 2) ;
+
+  @media (orientation: portrait) {
+    flex-direction: column-reverse;
+    align-items: flex-start;
+    row-gap: calc(var(--border-width) * 2) ;
+  }
+`;
```

Comment:
Please add new lines between components and end with ;

const Something = styled.div\`
    ....
\`;

Reply:
Thank you @tuj 
I have made the changes you suggested.
Wonder why prettier did not pick up on that.